app/controllers/main_controller.rb

```diff
@@ -3,17 +3,20 @@
 # controller for user model
 class MainController < ApplicationController
   def index
-    @users = User.all
-
-    @users.each do |user|
-      unless user.avatar.attached?
-        user.avatar.attach(io: File.open("storage/user-images/user-default.jpeg"),
-                           filename: "file.pdf")
-      end
-    end
+    @time = time { @users = User.all } # loads each attachment separately. don't know howto fix it.
+    # includes avatar_attachment doesn't make any difference. with_image_attachment either
   end
 
   def show
     @user = User.find(params[:id])
   end
+
+  private
+
+    # calculates time of data load
```

Comment:
It's temporary method while I'm trying to fix active storage blob load.

Reply:
Please move temp code to the separate branch out of this PR